src/CodecLz4.jl

```diff
@@ -21,7 +21,7 @@ struct LZ4Exception <: Exception
     msg::AbstractString
 end
 
-function Base.showerror(io::IO, ex::LZ4Exception, bt; backtrace=false)
```

Comment:
Why are we removing `bt` again?

Reply:
This was originally overriding https://github.com/JuliaLang/julia/blob/55e36cc308b66d3472990a06b2797f9f9154ea0a/base/errorshow.jl#L77, which deals with fetching and printing the backtrace as well as the error. This was blocking the backtrace from being shown which made debugging much harder.

We're now overriding https://github.com/JuliaLang/julia/blob/55e36cc308b66d3472990a06b2797f9f9154ea0a/base/errorshow.jl#L30 which is called within the longer form.